src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java

```diff
@@ -37,8 +34,10 @@
 
 import org.apache.commons.io.FilenameUtils;
 import org.jenkinsci.remoting.RoleChecker;
+import org.kohsuke.stapler.DataBoundSetter;
 
-import static com.google.api.client.http.HttpStatusCodes.STATUS_CODE_UNAUTHORIZED;
+import static com.google.api.client.http.HttpStatusCodes
+    .STATUS_CODE_UNAUTHORIZED;
```

Comment:
Import lines should probably just be ignored by checkstyle. The checkstyle config is already set up to ignore "import" lines. Perhaps just add `( static)?` to the regex.